src/main/java/gov/cabinetofice/gapuserservice/util/HelperUtils.java

```diff
@@ -57,6 +59,15 @@ public static String asJsonString(final Object obj) {
         }
     }
 
+    public static List<String> removeSquareBracketsAndTrim(List<String> inputList) {
+        List<String> cleanedList = new ArrayList<>();
+        for (String input : inputList) {
+            String cleanedString = input.replace("[", "").replace("]", "");
+            cleanedList.add(cleanedString.trim());
+        }
+        return cleanedList;
```

Comment:
I'm a sucker for streams
```suggestion
        return inputList.stream()
                .map(input -> input.replace("[", "").replace("]", "").trim())
                .collect(Collectors.toList());
```

Reply:
sure, happy with that too